packages/story-editor/src/app/media/utils/useProcessMedia.js

```diff
@@ -125,19 +126,21 @@ function useProcessMedia({
         copyResourceData({ oldResource, resource });
         updateOldTranscodedObject(resourceId, resource.id, 'source-video');
         deleteMediaElement({ id: resourceId });
-        if (
-          ['video', 'gif'].includes(resource.type) &&
-          !resource.local &&
-          !resource.posterId
-        ) {
+        if (resource.local) {
+          return;
+        }
+        if (['video', 'gif'].includes(resource.type) && !resource.posterId) {
           uploadVideoPoster(resource.id, resource.src);
         }
+        if ('video' === resource.type && resource.isMuted === null) {
+          updateVideoIsMuted(resource.id, resource.src);
+        }
+
         if (
-          'video' === resource.type &&
-          !resource.local &&
-          resource.isMuted === null
+          !Array.isArray(resource?.baseColor) ||
+          !resource?.baseColor.length
         ) {
-          updateVideoIsMuted(resource.id, resource.src);
+          updateBaseColor({ resource });
```

Comment:
Why are we calling `updateBaseColor` when optimizing an existing video?

Are there concerns the original video doesn't have a baseColor set?

Reply:
> Are there concerns the original video doesn't have a baseColor set?

Either the original video didn't have one set or opertizing the video change the video enough that the base color is different. It is a simple post about muting, this should be copied over, just being overly careful here.